examples/cartpole/README.md

```diff
@@ -3,32 +3,61 @@
 This example is a recreation of the classic [CartPole](https://www.gymlibrary.ml/environments/classic_control/cart_pole/)
 problem in [Webots](https://cyberbotics.com) as seen in [OpenAI's gym](https://gym.openai.com/).
```

Comment:
https://www.gymlibrary.ml is broken.
A website redirect points https://gym.openai.com to https://www.gymlibrary.dev/.
I think we can change it to https://gym.openai.com.
```suggestion
This example is a recreation of the classic [CartPole](https://gym.openai.com/environments/classic_control/cart_pole/)
problem in [Webots](https://cyberbotics.com) as seen in [OpenAI's gym](https://gym.openai.com/).
```

Reply:
Oops i tested all the links but didn't notice the redirection, thanks!